examples/ensemble/plot_voting_regressor.py

```diff
@@ -26,30 +32,56 @@
 from sklearn.linear_model import LinearRegression
 from sklearn.ensemble import VotingRegressor
 
-# Loading some example data
-X, y = datasets.load_boston(return_X_y=True)
-
+##############################################################################
 # Training classifiers
-reg1 = GradientBoostingRegressor(random_state=1, n_estimators=10)
-reg2 = RandomForestRegressor(random_state=1, n_estimators=10)
+# --------------------------------
+# First, we are going to load diabetes dataset and initiate gradient boosting
+# regressor, random forest regressor and linear regression. Next, we are going
+# to use each of them o build the voting regressor:
+
+X, y = datasets.load_diabetes(return_X_y=True)
+
+# Train classifiers
+reg1 = GradientBoostingRegressor(random_state=1)
+reg2 = RandomForestRegressor(random_state=1)
 reg3 = LinearRegression()
-ereg = VotingRegressor([('gb', reg1), ('rf', reg2), ('lr', reg3)])
+
 reg1.fit(X, y)
 reg2.fit(X, y)
 reg3.fit(X, y)
+
+ereg = VotingRegressor([('gb', reg1), ('rf', reg2), ('lr', reg3)])
 ereg.fit(X, y)
 
-xt = X[:20]
```

Comment:
If we are going to only plot the first 20, we can keep this line and only predict the first 20.